src/lib/mongoBackend/mongoGetSubscriptions.cpp

```diff
@@ -139,6 +139,7 @@ static void setSubject(Subscription* s, const BSONObj& r)
 static void setNotification(Subscription* subP, const BSONObj& r, const std::string& tenant)
 {
   // Attributes
+  // FIXME: use setStringVectorF
```

Comment:
This (an other similar FIXME) will be addressed in a very next PR.

NTC (just informative)